chat-client/src/client/tabs/tabFactory.ts

```diff
@@ -19,6 +20,8 @@ export class TabFactory {
     private history: boolean = false
     private export: boolean = false
     private agenticMode: boolean = false
+    private modelSelectionEnabled: boolean = false
+    private region: Region = 'us-east-1'
```

Comment:
is this always expected to be us-east-1? no other regions are currently supported?